cluster_utils/job.py

```diff
@@ -84,7 +84,19 @@ def generate_final_setting(self, paths):
         current_setting["id"] = self.id
         return current_setting
 
-    def generate_execution_cmd(self, paths):
+    def generate_execution_cmd(self, paths, cmd_prefix: Optional[str] = None):
+        """Generate the commands to execute the job.
+
+        Args:
+            paths: Dictionary containing relevant paths.
+            cmd_prefix: String that is prepended to the command that runs the user
+                script.  Can be used to wrap in in some cluster-system-specific command
```

Comment:
Hm. Do you use it consistently? With things like this, I am fine with it as long as it's consistent :)

Reply:
Yes, I think I'm consistent with it (it's pretty much in my muscle memory by now).  But of course it will likely not be consistent with parts written by other people.